polls/models.py

```diff
@@ -83,3 +83,38 @@ class Annotation(models.Model):
 
     def __str__(self):
         return f"Annotation by {self.user.username}"
+
+
+class ExperimentType(models.Model):
+    type_choices = (
+        ("2AFC", "2AFC"),
+        ("A/B", "A/B"),
+    )
+    type = models.CharField(max_length=10, choices=type_choices)
+
+    def __str__(self):
+        return f"{self.type}"
+
+
+class Experiment(models.Model):
+    name = models.CharField(max_length=50)
+    experiment_type = models.ForeignKey(ExperimentType, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class ExperimentTypeTaskPresentation(models.Model):
+    task_presentation = models.CharField(max_length=50)
```

Comment:
I thought this was a list of chars, like:

("XXY", "XYX") or ("X", "Y")

Reply:
He should consider adding options for the row.

Reply:
@DeFidelity can you explain what you mean?

Reply:
I mean since input types are already known to follow a pattern like ("XXY", "XYX") or ("X", "Y"), we can have options at the top where we just pass it to on the table as a variable to options. like it was done in the Annotation table

Reply:
Yes I thought of doing that but I thought that the options, that we will show, are dependent on the experiementType the user opts. So, in that case we should not hard code options. But if they will remain same throughout the system, we can add options here. Please let me know your thoughts on it.